account_security_extra/security/account_security_extra.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account.access_account_account" model="ir.model.access">
+        <field name="perm_write" eval="1"/>
+        <field name="perm_create" eval="1"/>
+    </record>
```

Comment:
Is it possible replacing this update with a new ACL that adds this functionality? I don't know if two ACLs to the same model and group could live together... If not, we should add to uninstall hook getting back original permissions